src/components/CompareResults/beta/RevisionSelect.tsx

```diff
@@ -0,0 +1,45 @@
+import SortIcon from '@mui/icons-material/Sort';
+import Box from '@mui/material/Box';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import SvgIcon from '@mui/material/SvgIcon';
+
+const revisionsOptions = [
+  'All revisions',
+  '6064a73fc99f',
+  'bc44c46861dd',
+  '3bebf28f32a7',
+];
+
+function RevisionSelect() {
+  return (
+    <Box data-testid={'revision-select'}>
+      <Select
+        disabled // To be removed
+        sx={{ width: 200 }}
+        defaultValue=''
+        displayEmpty
+        size='small'
+        renderValue={(value) => {
+          return (
+            <Box sx={{ display: 'flex', gap: 1 }}>
```

Comment:
Same here. I'd encourage keeping how we style individual components consistent throughout using the typestyle library.

For ex:

```
import { style } from 'typestyle';

const styles = {
  Box: style({
    display: 'flex',
    gap: '1px'
  }),
...
};
```